src/N3Util.js

```diff
@@ -34,6 +34,7 @@ export function inDefaultGraph(quad) {
 
 // Creates a function that prepends the given IRI to a local name
 export function prefix(iri, factory) {
+  if (isNamedNode(iri)) iri = iri.value;
   return prefixes({ '': iri }, factory)('');
```

Comment:
Might as well support literals too?
```suggestion
  return prefixes({ '': iri.value || iri }, factory)('');
```